guides/tutorial/part-1/orientation.md

```diff
@@ -161,11 +159,11 @@ super-rentals
 17 directories, 36 files
```

Comment:
```suggestion
17 répertoires, 36 fichiers
```

Reply:
Not sure we should translate the command output 🤔 is that supposed to depend on the OS language?